### ISSUE TITLE
Workflow list (command line) throws exceptions in SQLA if workflow is not started

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida_core#469]
When we store a workflow (simple or with sub-workflows), we don't start it and we ask for a workflow list, then under SQLA we get an exception.

E.g.
```

+ Workflow WFTestEmpty (pk: 179) is INITIALIZED [0h:00m:00s ago]
E
======================================================================
ERROR: test_workflow_info (aiida.backends.tests.workflows.TestWorkflowBasic)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/aiida/aiida-code/aiida_core/aiida/backends/tests/workflows.py", line 141, in test_workflow_info
    wf_cmd.workflow_list()
  File "/home/aiida/aiida-code/aiida_core/aiida/cmdline/commands/workflow.py", line 94, in workflow_list
    depth=parsed_args.depth))
  File "/home/aiida/aiida-code/aiida_core/aiida/orm/implementation/sqlalchemy/workflow.py", line 770, in get_workflow_info
    w.module_class, wf_labelstring, w.id, w.state.value, str_timedelta(
AttributeError: 'unicode' object has no attribute 'value'
```

**Comment by szoupanos** [comment]
Moreover the aiida.orm.implementation.sqlalchemy.workflow.Workflow#get_state doesn't seem to work.

It had to be changed from return `self.dbworkflowinstance.state.value` to return `self.dbworkflowinstance.state`.

This needs a better look (more than the 2 mins that I had time to look at it).
Maybe there was a schema change.

**Comment by szoupanos** [comment]
Fixed in #471
